core/src/main/resources/hudson/PluginManager/advanced_pl.properties

```diff
@@ -20,6 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 Advanced\ Settings=Ustawienia zaawansowane
```

Comment:
`core/src/main/resources/hudson/PluginManager/advanced.jelly` no longer contains an "Advanced Settings" with a capital S after #6783, so this line can be deleted:

```suggestion
```

Reply:
Then this is technical debt of mentioned PR and should be fixed as follow-up of them. Not by removing Polish translation but removing items from all localization.

Reply:
Indeed. For this reason, these suggestions were merely optional.